packages/client/src/components/BlogCard.jsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import { Box, Card, CardActionArea, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+import { prependStrapiURL } from '../utils';
+
+export function BlogCard({ id, tag, title, description, imgSource, author, date }) {
+  return (
+    <Card
+      variant="outlined"
+      sx={{ display: 'flex', flexDirection: 'column', height: '100%', maxWidth: 380, border: 0, borderRadius: 0 }}
+    >
+      <CardActionArea
+        component={Link}
+        to={`${id}/${title.toLowerCase().replace(/[^a-z0-9]+/g, '-')}`}
```

Comment:
Could we split this into utils since it recurs in BlogPost.jsx?

Reply:
Not entirely the same. The hovering actions are different. Plus, the margin/padding are much bigger in BlogCard. Contents displayed in the card are also a bit different.

Reply:
Sorry for being unclear - I meant only the `toLowerCase().replace(/[^a-z0-9]+/g, '-')` part!

Reply:
Got it. I can write a `titleToUrlPath` parser and put it under `util`